pages/about.html

```diff
@@ -0,0 +1,205 @@
+                                    fill="#A7A7A7" />
+                            </g>
+                            <defs>
+                                <clipPath id="clip0_187_3">
+                                    <rect width="201" height="63" fill="white" />
+                                </clipPath>
+                            </defs>
+                        </svg>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <footer class="container">
+            <a class="logo" href="/">Code 4 Easy</a>
+            <p>&copy; Copy right 2022</p>
+        </footer>
```

Comment:
As well, in order to follow the given design, please give the footer a background colour as in the design from Cindy Sheen in the about page;

Your design:
![image](https://user-images.githubusercontent.com/83596135/163475616-b2078fdb-91ac-4347-bfde-85c103aa1c83.png)

Given design:
![image](https://user-images.githubusercontent.com/83596135/163475947-944f9de9-5fe3-4699-af56-20afe5d225ea.png)

Reply:
Thanks 😊